src/vue/forms/TfaForm.vue

```diff
@@ -135,7 +137,10 @@ export default {
       loadFactors: vuexTypes.LOAD_FACTORS,
     }),
     async toggleFactor () {
-      if (this.$v.form.password.$invalid) return
+      if (this.$v.form.password.$invalid) {
+        this.$v.form.password.$touch()
```

Comment:
is not is should be done before the `this.$v.form.password.$invalid` check?

Reply:
Updated, `isFormValid` used